### DISCUSSION TITLE
My widget is not working for Wordpress and I'm not sure what to do lol

### DISCUSSION THREAD

**Discussion by WebsiteCoordinator** [code4recovery/12-step-meeting-list#1270] [Q&A] [answered]
Hello everyone! My name is Tiffany and I've been the website coordinator for my Area for about 2-3 years now and just came across a strange issue. When I go to update or add a meeting and select "Publish" it just sends it to my drafts and won't allow me to actually publish it to the site. I've tried updating everything and rebooting my computer and Wordpress, but nothing has helped. Any suggestions are appreciated.

**Comment by WebsiteCoordinator** [comment, marked as answer]
UPDATE: I may have solved the issue on my own.

**Reply by SIA-WEBSUP** [reply]
Please let us know if the issues has been resolved on its own.
